josh-works/corkboard
# File uploads

What does this PR do?
This adds the functionality for uploading files with paperclip and s3

Where should the reviewer start?
look at files changed - no major differences though

How should this be manually tested?
confirm it works

Any background context you want to provide?
used existing paperclip, added new configuration to use s3

What are the relevant tickets?
none

Screenshots (if appropriate)


Questions:


- Do Migrations Need to be ran?
no
- Do Environment Variables need to be set?
keys will need to be pushed to heroku separately (and distributed to the team)
- Any other deploy steps?
